ci-operator/step-registry/ipi/conf/gcp/service-account/ipi-conf-gcp-service-account-commands.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+PATCH="${SHARED_DIR}/service-account.yaml.patch"
+
+if [ -n "${COMPUTE_SERVICE_ACCOUNT}" ]; then
+  cat > "${PATCH}" << EOF
+compute:
+- platform:
+    gcp:
+      serviceAccount: ${COMPUTE_SERVICE_ACCOUNT}
```

Comment:
Once `COMPUTE_SERVICE_ACCOUNT` is set, how installer get its secret to complete the authentication with gcp api ?

Reply:
What I saw is, the compute/worker machines would be attached to the given `COMPUTE_SERVICE_ACCOUNT`. I guess the compute/worker machines would authenticate to GCP API using the attached service account which has the required permissions. Another example is, installing on a bastion host which is created with an attaching service account.

Reply:
Got it, good point.